schema/data/anchored/v0/classes/C06.jsonld

```diff
@@ -1,129 +1,51 @@
-    },
-    "regen:sectoralScope": {
-      "@container": "@list"
-    },
-    "regen:lookbackPeriod": {
-      "@type": "schema:Duration"
-    },
-    "regen:permanencePeriod": {
-      "@type": "schema:Duration"
-    },
-    "schema:itemListElement": {
-      "@container": "@list"
-    },
-    "regen:landOwnershipType": {
-      "@container": "@list"
-    },
-    "regen:bufferPoolAccounts": {
-      "@container": "@list"
-    },
-    "regen:eligibleActivities": {
-      "@container": "@list"
-    },
-    "regen:verificationMethods": {
-      "@container": "@list"
-    },
-    "regen:creditGenerationMethod": {
-      "@container": "@list"
-    },
-    "regen:additionalityAssessment": {
-      "@container": "@list"
-    }
+    "rfs": "https://framework.regen.network/schema/",
+    "schema": "http://schema.org/"
   },
   "schema:url": "https://www.registry.regen.network/crediting-protocols/ecometric---ghg-benefits-in-managed-crop-and-grassland-systems-credit-class",
   "schema:name": "GHG Benefits in Managed Crop and Grassland Systems Credit Class",
-  "regen:coBenefits": [
+  "rfs:hasCoBenefits": [
     {
-      "@id": "regen:ImprovedSoilStructureAndHydrologicalProperties",
-      "@type": "regen:ProjectBenefit",
-      "schema:name": ""
+      "rfs:impactType": "IMPROVED_SOIL_STRUCTURE",
+      "schema:name": "Improved Soil Structure & Hydrological Properties"
     },
     {
-      "@id": "regen:ReductionsInExternalInputs",
-      "@type": "regen:ProjectBenefit",
+      "rfs:impactType": "REDUCED_FERTILIZER_AMENDMENTS",
       "schema:name": "Reduced Fertilizer Usage"
     }
   ],
-  "regen:measuredGHGs": [
-    "Carbon Dioxide (CO2)"
-  ],
   "schema:description": "This Credit Class and its supporting measuring and monitoring methodology, focuses on quantifying increases in soil organic carbon sequestration and storage in regenerative cropping and managed grassland agricultural systems.",
-  "regen:creditingTerm": "P10Y",
-  "regen:ecosystemType": [
-    "Cropland",
-    "Grassland",
-    "Pastureland",
-    "Shrubland"
-  ],
-  "regen:primaryImpact": {
-    "@id": "regen:SoilOrganicCarbon",
+  "rfs:hasCreditingPeriod": "P10Y",
+  "rfs:eligibleEnvironmentTypes": [
+    "CROPLAND",
+    "GRASSLAND",
+    "PASTURE",
+    "SHRUBLAND"
+  ],
+  "rfs:hasPrimaryImpact": {
+    "rfs:impactType": "INCREASED_CARBON_SEQUESTRATION_STORAGE",
     "schema:name": "Soil Organic Carbon"
   },
-  "regen:sectoralScope": [
-    "Agriculture, Forestry, and Other Land Use (AFOLU)"
-  ],
-  "regen:lookbackPeriod": "-P10Y",
-  "regen:permanencePeriod": "P10Y",
-  "regen:landOwnershipType": [
-    "Public",
-    "Private",
-    "Tribal"
-  ],
-  "regen:bufferPoolAccounts": [
+  "rfs:hasPermanencePeriod": "P10Y",
+  "rfs:hasBufferPoolAccounts": [
     {
       "schema:name": "Blaston Estate Project Specific Buffer Pool",
-      "regen:walletAddress": "",
-      "regen:poolAllocation": "20%"
+      "rfs:walletAddress": "",
+      "rfs:poolAllocation": "20%"
```

Comment:
this is 10%

 - 10% of the credits go to the Crediting Term Buffer Pool: `regen19rrgcsw8gu8c3rthucqnf6nyyg6q9pq79tt60pvahfsnfu4p5hrs9z2n3j`
- 10% of the credits go to the Permanence Reversal Buffer: `regen15lch5glk7deu9tk8wrcfcup4tdpz2l8zhhqn4r2zzsr46dfv849qqfpu9m`